test/puppetlabs/puppetdb/mq_test.clj

```diff
@@ -109,8 +110,34 @@
       (is (= size-bytes (.. broker (getSystemUsage) (getTempUsage) (getLimit)))))))
 
 (deftest test-jmx-enabled
-  (svc-utils/without-jmx
+  (without-jmx
    (svc-utils/with-puppetdb-instance
      (is (thrown-with-msg? clojure.lang.ExceptionInfo
                            #"status 404"
                            (svc-utils/current-queue-depth))))))
+
+(deftest queue-operations
```

Comment:
I think this test should be broken up into separate tests for the various operations. With all the operations being tested at once, I find it hard to follow and separate what's being tested in my head.